src/main/resources/assets/immersiveengineering/manual/en_us/conveyors.txt

```diff
@@ -9,7 +9,7 @@ The panel on its side allows attaching a lever, but a redstone signal may be inp
 <&conveyor_dropper_recipe>The Dropping Conveyor Belt is quite simply a conveyor with a hatch in the middle, allowing it to drop items that pass over it downwards.
 Items will be inserted into inventories below or simply dropped if there is an empty space below the conveyor.
 The hatch can be shut with a redstone signal, when desired.<np>
-<&conveyor_extract_recipe>The Extracting Conveyor Belt possesses the ability to extract items from an inventory in a similar fashion to a Hopper. Unlike the latter however, it possesses no internal inventory and will place extracted items onto the belt.
+<&conveyor_extract_recipe>The Extracting Conveyor Belt possesses the ability to extract items from an inventory in a similar fashion to a Hopper. Unlike the latter however, it possesses no internal inventory and will place extracted items onto the belt. Can be stopped from pulling with a redstone signal.
```

Comment:
This change should not be in this pullrequest, it has absolutely nothing to do with the original issue.